testing/src/main/java/io/grpc/internal/testing/AbstractTransportTest.java

```diff
@@ -1265,6 +1338,28 @@ private void doPingPong(MockServerListener serverListener) throws InterruptedExc
     ServerStream serverStream = serverStreamCreation.stream;
     ServerStreamListener mockServerStreamListener = serverStreamCreation.listener;
 
+    verify(mockServerStreamListener, never()).messageRead(any(InputStream.class));
```

Comment:
Actually sending messages seems unnecessary. This is mainly just making sure the HTTP/2 connection is still working.